lib/ruby-lint/analysis/loop_keywords.rb

```diff
@@ -57,7 +58,7 @@ class LoopKeywords < Base
       # @param [RubyLint::AST::Node] node
       #
       def verify_keyword(keyword, node)
-        if current_scope.type != :block and !@allow_keyword
+        if current_scope.type != :block && [0, nil].include?(@allow_keyword)
```

Comment:
@mvidner Please submit that to the same PR (or a new PR containing all the changes), I don't want to pick commits from random places.